test/examples_test.go

```diff
@@ -33,18 +34,16 @@ import (
 	knativetest "knative.dev/pkg/test"
 )
 
-var (
-	pipelineRunTimeout = 10 * time.Minute
-)
+const pipelineRunTimeout = 10 * time.Minute
 
-const (
-	DEFAULT_KO_DOCKER_REPO = `gcr.io\/christiewilson-catfactory`
-	DEFAULT_NAMESPACE      = `namespace: default`
+var (
+	defaultKoDockerRepoRE = regexp.MustCompile("gcr.io/christiewilson-catfactory")
+	defaultNamespaceRE    = regexp.MustCompile("namepspace: default")
```

Comment:
hm there's a typo there, which makes me wonder if this isn't being used the way we think it is?

Reply:
Pretty sure this was why the tests broke. Yay tests!